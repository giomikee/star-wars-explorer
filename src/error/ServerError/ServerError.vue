<template>
  <v-container>
    <v-icon class="text-h1 w-100" color="error" icon="mdi-server-off" />
    <h2 class="text-center">A technical issue has occurred. Please try again later.</h2>
  </v-container>
</template>

<script lang="ts">
  export default { name: 'ServerError' }
</script>
