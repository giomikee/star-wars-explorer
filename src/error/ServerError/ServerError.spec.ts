import { shallowMount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import ServerError from './ServerError.vue'

const createWrapper = () => shallowMount(ServerError)

describe('Given ServerError component', () => {
  describe('When component is rendered', () => {
    it('Then it should match component snapshot', () => {
      const { element } = createWrapper()

      expect(element).toMatchSnapshot()
    })
  })
})
