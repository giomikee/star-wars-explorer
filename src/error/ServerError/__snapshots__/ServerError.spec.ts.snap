// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given ServerError component > When component is rendered > Then it should match component snapshot 1`] = `
<div
  class="v-container v-locale--is-ltr"
>
  <v-icon-stub
    class="text-h1 w-100"
    color="error"
    disabled="false"
    end="false"
    icon="mdi-server-off"
    size="default"
    start="false"
    tag="i"
  />
  <h2
    class="text-center"
  >
    A technical issue has occurred. Please try again later.
  </h2>
</div>
`;
